src/app/components/OnChainReceive/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { CaretLeftIcon } from "@bitcoin-design/bitcoin-icons-react/filled";
+import Header from "@components/Header";
+import { Trans, useTranslation } from "react-i18next";
+import { useNavigate } from "react-router-dom";
+import Button from "~/app/components/Button";
+import Container from "~/app/components/Container";
+import IconButton from "~/app/components/IconButton";
+
+export default function OnChainReceive() {
+  const navigate = useNavigate();
+  const { t } = useTranslation("translation", {
+    keyPrefix: "on_chain",
+  });
+
+  return (
+    <div className=" flex flex-col overflow-y-auto no-scrollbar">
+      <Header
+        title={t("title")}
+        headerLeft={
+          <IconButton
+            onClick={() => {
+              navigate(-1);
+            }}
+            icon={<CaretLeftIcon className="w-4 h-4" />}
+          />
+        }
+      />
+      <div className="pt-4">
+        <Container justifyBetween maxWidth="sm">
+          <div className="text-center dark:text-neutral-200 h-full flex flex-col justify-center items-center">
+            <div className="mb-8">
+              <p>
+                <Trans
+                  i18nKey={"instructions1"}
+                  t={t}
+                  components={[<strong key="instruction1-strong"></strong>]}
```

Comment:
I think `-strong` should be removed from the key name. It is just decoration, not related to the translation.